WebDriverAgentLib/Categories/XCUIElement+FBIsVisible.m

```diff
@@ -181,13 +211,24 @@ - (BOOL)fb_isVisible
     // However, upside-down case cannot be covered this way, which is not important for Appium
     midPoint = FBInvertPointForApplication(midPoint, appFrame.size, FBApplication.fb_activeApplication.interfaceOrientation);
   }
-  XCElementSnapshot *hitElement = [self hitTest:midPoint];
-  if (nil != hitElement && (self == hitElement || [ancestors containsObject:hitElement])) {
-    return [self fb_cacheVisibilityWithValue:YES forAncestors:ancestors];
+  XCAccessibilityElement *hitElement = [self elementAtPoint:midPoint];
```

Comment:
In my tests just getting rid of the `hitElement` logic here entirely fixes the problem and all the rest of the tests work.

Reply:
visibility detection was always a pain because there are many corner cases. Perhaps, these cases just don't popup in that set of tests :(

Reply:
Yup. Hopefully this works.